### ISSUE TITLE
 Drop dependency on cryptonite-openssl package

### ISSUE THREAD

**Issue by dcoutts** [input-output-hk/cardano-node#247]
The CLI tools in this repo currently rely on the [cryptonite-openssl] package for getting entropy for the purpose of creating crypto key pairs. This should be replaced by the equivalent functionality from the [cryptonite] package.

The OpenSSL version is designed to allow a high volume of random entropy to be generated in a reasonable way (e.g. for use in TLS). This is not a requirement we have, since we only use it in CLI tools for generating key pairs, which is very low volume. So it is acceptable to use the functionality in `cryptonite` which just gets randomness direct from the OS (`/dev/random` on unix, equiv on Windows).

[cryptonite-openssl]: http://hackage.haskell.org/package/cryptonite-openssl
[cryptonite]: http://hackage.haskell.org/package/cryptonite

**Comment by dcoutts** [comment]
Note that this also depends on #242 to fully drop the dependency.

**Comment by dcoutts** [comment]
Note also that this is used in one place in `cardano-ledger` but only to help the CLI tools for key gen. See `crypto/src/Cardano/Crypto/Random.hs`.

**Comment by kevinhammond** [comment]
Fix this if not already done?

**Comment by Jimbo4350** [comment]
Resolved with the linked PRs.
